include/cxxopts.hpp

```diff
@@ -884,7 +884,7 @@ namespace cxxopts
       ("--([[:alnum:]][-_[:alnum:]]+)(=(.*))?|-([[:alnum:]]+)");
 
     std::basic_regex<char> option_specifier
-      ("(([[:alnum:]]),)?([[:alnum:]][-_[:alnum:]]*)?");
+      ("(([[:alnum:]]),)?[ ]*([[:alnum:]][-_[:alnum:]]*)?");
```

Comment:
"\s" will match any whitespace character including "\t" and "\n".
I just opted for the tightest implementation for the requirement of allowing spaces.
I don't mind sending another pull request with \s.